packages/@o3r/workspace/schematics/application/index.ts

```diff
@@ -6,6 +6,7 @@ import type { PackageJson } from 'type-fest';
 import { RunSchematicTask } from '@angular-devkit/schematics/tasks';
 import { type Schema as ApplicationOptions } from '@schematics/angular/application/schema';
 import { Style } from '@schematics/angular/application/schema';
+import { updateProjectTsConfig } from '../rule-factories';
```

Comment:
The _rule-factories_ does not exist in _o3r/workspace._ It should be created. 
Ref: my previous comment